Testing/Functional/Products/Products.Tests.ps1

```diff
@@ -249,8 +249,9 @@ Describe "Policy Checks for <ProductName>"{
                 SetConditions -Conditions $Preconditions.ToArray()
                 RunScuba
             }
-            elseif ('SCuBACached' -eq $TestDriver){
-                Write-Debug "Driver: SCuBACached"
+            # Cmdlet names should match Test Plan drivers e.g. Driver should be ScubaCached instead of RunCached to be aligned with the refactor of Scuba specific Cmdlets
```

Comment:
(nit) "the refactor" is a point in time comment that will not make sense to someone a few years down the road (which refactor? we did a refactor? etc).  Instead, I would add a comment at the top of this if statement (e.g. line 222) that says something like:

\# Select which TestDriver to use for a given test plan.  TestDriver names (e.g. RunScuba, ScubaCached) must
\# match exactly (including case) the ones used in TestPlans.